### PR TITLE
Added a -label-on-resolved flag.

### PR DESCRIPTION
**Pull request by cjyar**
This addresses #37.

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/alertmanager-github-receiver/38)

### CODE REVIEW

File: issues/handler_test.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func TestListHandler_ServeHTTP(t *testing.T) {
 			expectedStatus: http.StatusOK,
 			listClient: &fakeClient{
 				issues: []*github.Issue{
-					&github.Issue{
```
Comment (cjyar) [inline issues/handler_test.go:50]: I know it seems unrelated, but this cleans up a lint warning I was getting from go1.14.

File: issues/local/local.go [hunk 2]
```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 //////////////////////////////////////////////////////////////////////////////
 
-// Package memory provides in memory operations on GitHub issues.
+// Package local provides in memory operations on GitHub issues.
```
Comment (cjyar) [inline issues/local/local.go:16]: Just fixing another lint warning here.

**Comment by cjyar** [comment]
@stephen-soltesz Ready for re-review.

**Comment by cjyar** [comment]
Now I'm embarrassed. It looks like the GitHub API for deleting a label from an issue isn't idempotent, so this code doesn't actually work. :(

I was deploying it in our test instance while you were merging; too slow on this end.

I'll work on a fix.

**Comment by stephen-soltesz** [comment]
Oh no! Thanks for noticing quickly.

**Pull request merged**